.github/workflows/ci.yml

```
@@ -106,7 +106,7 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.11"]
-        nautobot-version: ["2.0.0"]
+        nautobot-version: ["2.4.0"]
```

Since we're setting the minimum version to 2.4.2 we might as well update ci to test against that

```suggestion
        nautobot-version: ["2.4.2"]
```